sqlite-android/src/main/jni/sqlite/CursorWindow.cpp

```diff
@@ -54,6 +57,7 @@ status_t CursorWindow::create(const std::string& name, size_t size, CursorWindow
         *outWindow = window;
         return OK;
     }
+    delete window;
```

Comment:
This was leaked before this as far as I can say though is unlikely to happen, perhaps can be proposed as a separate patch.